synapse/tests/test_model_person.py

```diff
@@ -1,7 +1,25 @@
+import synapse.lib.tufo as s_tufo
+import synapse.lib.types as s_types
+
 from synapse.tests.common import *
 
 class PersonTest(SynTest, ModelSeenMixin):
 
+    def test_model_ps_has(self):
+        with self.getRamCore() as core:
+            person_guid = 32 * '0'
+            person_tufo = core.formTufoByProp('ps:person', person_guid, name='Kenshoto,Invisigoth')
+            personval = person_tufo[1].get('ps:person')
+            fqdn_tufo = core.formTufoByProp('inet:fqdn', 'vertex.link')
+
+            node = core.formTufoByProp('ps:has', (personval, ('inet:fqdn', 'vertex.link')))
```

Comment:
I'll expand this test to include the rest of the forms we are removing as I remove them